src/common/SIRF.py

```diff
@@ -55,7 +55,7 @@ def __del__(self):
         print("SIRF.DataContainer __del__ with handle {}.".format(self.handle))
         if self.handle is not None:
             pyiutil.deleteDataHandle(self.handle)
-    @abc.abstractmethod
+#    @abc.abstractmethod
```

Comment:
Why don't we want this to be abstract anymore?

Reply:
`tests_syn.py` fails if this method is abstract